scripts/jenkins-build.bat

```diff
@@ -68,11 +68,32 @@ cmake -G "Ninja" ^
 cmake --build . || exit /b
 cmake --build . --target install || exit /b
 
+REM Build Java
 cd "%WORKSPACE%"
+cd source
+call mvn clean install
```

Comment:
If "mvn" is actually calling an "mvn.bat" then you'll likely need to use `call`.  Do you still need the `|| exit /b` in this case for correct error handling?